src/execution_plan/optimizations/utilize_indices.c

```
@@ -95,8 +95,17 @@ static bool _applicableInExpression
 		SIValue list = SI_NullVal();
 		AR_EXP_ReduceToScalar(rhs, true, &list);
 
+		// range() is intentionally marked as non-reducible; for IN index
```

Cppcheck (2.20.0)

[error] Common realloc mistake

(memleakOnRealloc)